### ISSUE TITLE
Allow importing external vocabularies as closed vocabularies in editor

### ISSUE THREAD

**Issue by twagoo** [clarin-eric/component-registry-front-end#63]
https://trac.clarin.eu/wiki/CMDI%201.2/Vocabularies

**Comment by twagoo** [comment]
According to Menzo, the SKOS API does not allow for retrieval of all items in a vocabulary. The OAI endpoint might provide a solution for this. Also, ask Olha, she's up-to-date with the latest developments.

**Comment by twagoo** [comment]
Concepts can be obtained via `{api}/find-concepts?q=prefLabel:*&conceptScheme={schemeUri}[&format=json]`

e.g. [find-concepts?q=prefLabel:*&conceptScheme=http://openskos.meertens.knaw.nl/iso-639-3&format=json](http://145.100.58.79/clavas/public/api/find-concepts?q=prefLabel:*&conceptScheme=http://openskos.meertens.knaw.nl/iso-639-3&format=json)

**Comment by twagoo** [comment]
Request only these fields (`fl` query parameter can be used, see [Solr syntax guide](https://cwiki.apache.org/confluence/display/solr/Common+Query+Parameters)):
- __uri__ (for @ConceptLink)
- the specified value property for the vocabulary, combing `@ValueProperty` and `@ValueLanguage` (default: __prefLabel@en__)

Results in vocabulary item
```
{
 '$': item[vocab.valueProperty+'@'+vocab.valueLanguage]
 'conceptLink': item.uri
}
```

**Comment by twagoo** [comment]
Implemented as of 214ae447401ea123e9c974d4129a85b3ba3d41e5
Follow-up ticket #86
